src/xdocs/google_style.xml

```diff
@@ -1408,6 +1408,43 @@
                     test</a>
                 </td>
               </tr>
+              <tr>
+                <td><a name="4.8.5.3"/>
+                  <a href="#4.8.5.3">
+                    <span class="wrapper inline">
+                      <img src="images/anchor.png" alt=""/>
+                    </span>
+                  </a>
+                </td>
+                <td>
+                  <a href="styleguides/google-java-style-20220203/javaguide.html#s4.8.5.3-method-annotation-style">
+                    4.8.5.3 Methods And Constructors Annotations</a>
+                </td>
+                <td>
+                  <span class="wrapper inline">
+                    <img src="images/ok_green.png" alt="" />
+                  </span>
+                  <a href="checks/annotation/annotationlocation.html#AnnotationLocation">
+                    AnnotationLocation</a>
+                  <br />
+                  <br />
+                  <span class="wrapper inline">
+                    <img src="images/ok_green.png" alt="" />
+                  </span>
+                  <a href="checks/javadoc/invalidjavadocposition.html#InvalidJavadocPosition">
+                    InvalidJavadocPosition</a>
```

Comment:
why javadoc Check appears here ?
Chapter does not care on javadoc.

Reply:
it does, here's the its usage:

https://github.com/checkstyle/checkstyle/blob/f82714823ef21f43fc3543de5706690ead752f0f/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4853methodsandconstructorsannotations/InputMethodsAndConstructorsAnnotations.java#L28-L33

https://github.com/checkstyle/checkstyle/blob/f82714823ef21f43fc3543de5706690ead752f0f/src/it/java/com/google/checkstyle/test/chapter4formatting/rule4853methodsandconstructorsannotations/MethodsAndConstructorsAnnotationsTest.java#L53-L59

Reply:
There is another chapter in style guide that demands proper locations for all javadocs. But wording of this chapter is not mentioned about javadoc anything.
Im this table we map wording to Checks, to let anyone of us in future look at this table and see how each phrase is related to used Checks.

Reply:
Please share words from chapter that you think vis about location of javadoc

Reply:
From Filed Annotations' documentation:

> The rules for annotations on method and constructor declarations are the same as the [previous section](https://google.github.io/styleguide/javaguide.html#s4.8.5.2-class-annotation-style). Example:

here the previous section is Class Annotations. 

From Class Annotations' documentation:

> Annotations applying to a class appear immediately after the documentation block, and each annotation is listed on a line of its own (that is, one annotation per line).

This does talks about the documentation ( i.e javadoc ). We've used the InvalidJavadocPosition in that section so we're using at here as this section has same rules as it.

Reply:
I see it now, thanks a lot !